inc/admin.php

```diff
@@ -78,6 +88,7 @@ public function inline_bootstrap_script() {
 		
 		</style>
 		<script type="application/javascript">
+					document.documentElement.className += "optimole_has_js";
```

Comment:
I think we need a space before the class, in case that the className already contains one, it will concat them rather than adding a new class.  
```suggestion
					document.documentElement.className += " optimole_has_js";
```